### PR TITLE
Fix #658: Lowfi admin controls part 2 basic admincontrols activity

### LINKED ISSUE #658: Lowfi Admin Controls (Part 2):  Introduce basic AdministratorControlsActivity
Mock: https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/71345025-f19e-4cac-a63f-e94442540e8f/Settings-6

All these options should be visible in admin profiles.

## Checklist
- [ ] Finish `AdministratorControlsActivity` (not its child activity)
- [ ] Test cases that everything is displayed correctly

### PR DESCRIPTION
**Pull request by Luffy18346**
## Explanation
Fix #658 Lowfi Admin Controls part 2: Introduce basic AdministratorControlsActivity

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Luffy18346** [comment]
Here are the screenshots of the AdministratorControlsActivity.

Screenshot_2020-02-23-13-03-33-68_092c5771dc9bb06bcb5e5b6a9dadbf56 1

Screenshot_2020-02-23-13-03-38-59_092c5771dc9bb06bcb5e5b6a9dadbf56 1

**Comment by Luffy18346** [comment]
@rt4914 I have used paddingTop = 20dp and paddingBottom = 20dp instead of 18, 19, 28dp, etc. given in the mock, to keep uniform spaces at top and bottom in all views. Is this ok or should I change them according to the mock?

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
+import androidx.drawerlayout.widget.DrawerLayout
 import org.oppia.app.R
+import org.oppia.app.activity.ActivityScope
+import org.oppia.app.drawer.NavigationDrawerFragment
 import javax.inject.Inject
 
 /** The presenter for [AdministratorControlsActivity]. */
+@ActivityScope
 class AdministratorControlsActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
+  private var navigationDrawerFragment: NavigationDrawerFragment? = null
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt]: You can update this to 
`private lateinit var navigationDrawerFragment: NavigationDrawerFragment` and once this is done, you will need t to change, line 34 too.

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt [hunk 2]
```diff
@@ -16,6 +25,18 @@ class AdministratorControlsActivityPresenter @Inject constructor(private val act
...
 
+  private fun setUpNavigationDrawer() {
+    val toolbar = activity.findViewById<View>(R.id.administrator_controls_activity_toolbar) as Toolbar
+    activity.setSupportActionBar(toolbar)
+    activity.supportActionBar!!.setDisplayShowHomeEnabled(true)
+    navigationDrawerFragment =
+      activity.supportFragmentManager.findFragmentById(R.id.administrator_controls_activity_fragment_navigation_drawer) as NavigationDrawerFragment
+    navigationDrawerFragment!!.setUpDrawer(
+      activity.findViewById<View>(R.id.administrator_controls_activity_drawer_layout) as DrawerLayout,
+      toolbar, 0
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt]: Update to `toolbar, /* menuItemId = */ 0`, to make it clear to understand.

File: app/src/main/java/org/oppia/app/drawer/NavigationDrawerFooterViewModel.kt [hunk 3]
```diff
@@ -1,9 +1,9 @@
 package org.oppia.app.drawer
 
 import androidx.databinding.ObservableField
-import androidx.lifecycle.ViewModel
 import org.oppia.app.viewmodel.ObservableViewModel
 import javax.inject.Inject
+import androidx.lifecycle.ViewModel
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/drawer/NavigationDrawerFooterViewModel.kt]: Undo this change.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Blocking this PR for a day, to confirm on the approach, mean while you can fix changes in this PR and also, you can work on `LogOut` functionality from this https://github.com/oppia/oppia-android/issues/659

But it should be in a separate PR.

**Comment by rt4914** [comment]
> @rt4914 I have used paddingTop = 20dp and paddingBottom = 20dp instead of 18, 19, 28dp, etc. given in the mock, to keep uniform spaces at top and bottom in all views. Is this ok or should I change them according to the mock?

@Luffy18346 I have confirmed with team, we should implement this UI using recyclerview.

File: app/src/main/java/org/oppia/app/drawer/NavigationDrawerFooterViewModel.kt (hunk 3 above)
Comment (Luffy18346) [reply app/src/main/java/org/oppia/app/drawer/NavigationDrawerFooterViewModel.kt]: Done.

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt (hunk 2 above)
Comment (Luffy18346) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt]: Done.

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt (hunk 1 above)
Comment (Luffy18346) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt]: Done

**Comment by Luffy18346** [comment]
> > @rt4914 I have used paddingTop = 20dp and paddingBottom = 20dp instead of 18, 19, 28dp, etc. given in the mock, to keep uniform spaces at top and bottom in all views. Is this ok or should I change them according to the mock?
> 
> @Luffy18346 I have confirmed with team, we should implement this UI using recyclerview.

Done.

**Comment by Luffy18346** [comment]
@rt4914 I have pushed latest code (along with merge from upstream/develop) in this branch but its till giving me conflicts as u can see above. I am trying to resolve them in file but it is giving me an error "This page is out of date. Start over", every time I resolve conflicts.

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import javax.inject.Inject
+
+/** [ViewModel] for [AdministratorControlsFragment]. */
+@FragmentScope
+class AdministratorControlsViewModel @Inject constructor() : ViewModel() {
+  private val itemViewModelList: ObservableList<AdministratorControlsItemViewModel> = ObservableArrayList()
+
+  fun processAdministratorControlsList(): ObservableList<AdministratorControlsItemViewModel> {
+    itemViewModelList.add(AdministratorControlsGeneralProfileViewModel())
+
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt]: Remove blank line.

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    itemViewModelList.add(AdministratorControlsDownloadPermissionsViewModel())
+
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt]: Remove blank line.

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+import org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel.AdministratorControlsDownloadPermissionsViewModel
+import org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel.AdministratorControlsGeneralProfileViewModel
+import org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel.AdministratorControlsItemViewModel
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.app.viewmodel.ObservableArrayList
+import javax.inject.Inject
+
+/** [ViewModel] for [AdministratorControlsFragment]. */
+@FragmentScope
+class AdministratorControlsViewModel @Inject constructor() : ViewModel() {
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt]: Make this an `ObservableViewModel` instead of `ViewModel`

**Review by rt4914 — CHANGES_REQUESTED** [review]
It is good but I think the division of different layout is incorrect.

Instead of this we should have divided this it something like this:

`General` 
`ProfileManagement`
`Download Permissions`
`App Information`
`Account Actions`

So basically, `administrator_controls_general_profile_view.xml` and `administrator_controls_application_settings_view.xml` will get divided into two parts.

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt (hunk 6 above)
Comment (Luffy18346) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt]: Done

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt (hunk 5 above)
Comment (Luffy18346) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt]: Done

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt (hunk 4 above)
Comment (Luffy18346) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt]: Done

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsFragmentPresenter.kt [hunk 7]
```diff
@@ -3,19 +3,107 @@ package org.oppia.app.administratorcontrols
...
+  private lateinit var binding: AdministratorControlsFragmentBinding
+  private lateinit var linearLayoutManager: LinearLayoutManager
+
   fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?): View? {
-    val binding = AdministratorControlsFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)
-    binding.let {
-      it.lifecycleOwner = fragment
+    binding = AdministratorControlsFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)
+
+    val viewModel = getAdministratorControlsItemViewModel()
```
Comment (veena14cs) [inline app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsFragmentPresenter.kt]: Remove this line if `viewModel` is not getting used anywhere.

**Review by veena14cs — APPROVED** [review]
One nit change. Otherwise looks good.

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsFragmentPresenter.kt (hunk 7 above)
Comment (Luffy18346) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsFragmentPresenter.kt]: done.

**Review by veena14cs — APPROVED** [review]
LGTM

File: app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAccountActionsViewModel.kt [hunk 8]
```diff
@@ -0,0 +1,4 @@
+package org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel
+
+/**  view model for the recycler view in [AdministratorControlsFragment]. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAccountActionsViewModel.kt]: /** [ViewModel] the recycler view in [AdministratorControlsFragment]. */

File: app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAppInformationViewModel.kt [hunk 9]
```diff
@@ -0,0 +1,4 @@
+package org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel
+
+/** Application settings view model for the recycler view in [AdministratorControlsFragment]. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAppInformationViewModel.kt]: Ditto

File: app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsGeneralViewModel.kt [hunk 10]
```diff
@@ -0,0 +1,4 @@
+package org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel
+
+/** General account details view model for the recycler view in [AdministratorControlsFragment]. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsGeneralViewModel.kt]: Ditto

File: app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsProfileViewModel.kt [hunk 11]
```diff
@@ -0,0 +1,4 @@
+package org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel
+
+/** Profile management view model for the recycler view in [AdministratorControlsFragment]. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsProfileViewModel.kt]: Ditto

File: app/src/main/res/layout/administrator_controls_account_actions_view.xml [hunk 12]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
```
Comment (rt4914) [inline app/src/main/res/layout/administrator_controls_account_actions_view.xml:5]: Remove this. This is not getting used.

File: app/src/main/res/layout/administrator_controls_app_information_view.xml [hunk 13]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
```
Comment (rt4914) [inline app/src/main/res/layout/administrator_controls_app_information_view.xml:5]: Remove this. This is not getting used.

File: app/src/main/res/layout/administrator_controls_general_view.xml [hunk 14]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
```
Comment (rt4914) [inline app/src/main/res/layout/administrator_controls_general_view.xml:5]: Remove this. This is not getting used.

File: app/src/main/res/layout/administrator_controls_profile_view.xml [hunk 15]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
```
Comment (rt4914) [inline app/src/main/res/layout/administrator_controls_profile_view.xml:5]: Remove this. This is not getting used.

File: app/src/main/res/values/strings.xml [hunk 16]
```diff
@@ -223,4 +224,19 @@
   <string name="add_profile_required">*Required</string>
   <string name="back_button">Back Button</string>
   <string name="next">Next</string>
+  <!-- AdministratorControlsFragment. -->
```
Comment (rt4914) [inline app/src/main/res/values/strings.xml]: Remove full stop from the end `<!-- AdministratorControlsFragment -->`

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt [hunk 17]
```diff
@@ -82,6 +87,53 @@ class AdministratorControlsActivityTest {
...
+  }
+
+  @Test
+  fun testAdministratorControlsFragment_loadFragment_displayApplicationSettings() {
+    ActivityScenario.launch<AdministratorControlsActivity>(createAdministratorControlsActivityIntent(0)).use {
+      onView(withId(R.id.administrator_controls_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(2))
+      onView(atPositionOnView(R.id.administrator_controls_list, 2, R.id.app_information_text_view)).check(matches(isDisplayed()))
+      onView(atPositionOnView(R.id.administrator_controls_list, 2, R.id.app_version_text_view)).check(matches(withText("App Version")))
+      onView(atPositionOnView(R.id.administrator_controls_list, 2, R.id.account_actions_text_view)).check(matches(isDisplayed()))
+      onView(atPositionOnView(R.id.administrator_controls_list, 2, R.id.log_out_text_view)).check(matches(withText("Log Out")))
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt]: The strings in this file should be fetched from `strings.xml` file.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks. Suggested some final changes.

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt [hunk 18]
```diff
@@ -82,6 +87,53 @@ class AdministratorControlsActivityTest {
     }
   }
 
+  @Test
+  fun testAdministratorControlsFragment_loadFragment_displayGeneralAndProfileManagement() {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt:91]: Test cases are failing because of new implemenation.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Test cases are failing because of new implementation. Earlier they were passing. You just need to update the indexes.

File: app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAccountActionsViewModel.kt (hunk 8 above)
Comment (Luffy18346) [reply app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAccountActionsViewModel.kt]: Done

File: app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAppInformationViewModel.kt (hunk 9 above)
Comment (Luffy18346) [reply app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAppInformationViewModel.kt]: Done

File: app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsGeneralViewModel.kt (hunk 10 above)
Comment (Luffy18346) [reply app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsGeneralViewModel.kt]: Done

File: app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsProfileViewModel.kt (hunk 11 above)
Comment (Luffy18346) [reply app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsProfileViewModel.kt]: Done

File: app/src/main/res/layout/administrator_controls_profile_view.xml (hunk 15 above)
Comment (Luffy18346) [reply app/src/main/res/layout/administrator_controls_profile_view.xml]: Done

File: app/src/main/res/values/strings.xml (hunk 16 above)
Comment (Luffy18346) [reply app/src/main/res/values/strings.xml]: Done

File: app/src/main/res/layout/administrator_controls_account_actions_view.xml (hunk 12 above)
Comment (Luffy18346) [reply app/src/main/res/layout/administrator_controls_account_actions_view.xml:5]: @rt4914 I cannot remove this `<data>` tag and `ViewModel `implementation inside it because otherwise it will give error while registering it in `MultiTypeBuilder ` in `setViewModel ` parameter as there won't be any `ViewModel ` class, associated with the corresponding `ViewBinding `class.

File: app/src/main/res/layout/administrator_controls_app_information_view.xml (hunk 13 above)
Comment (Luffy18346) [reply app/src/main/res/layout/administrator_controls_app_information_view.xml:5]: I cannot remove for above mentioned reason

File: app/src/main/res/layout/administrator_controls_general_view.xml (hunk 14 above)
Comment (Luffy18346) [reply app/src/main/res/layout/administrator_controls_general_view.xml:5]: I cannot remove for above mentioned reason

File: app/src/main/res/layout/administrator_controls_profile_view.xml (hunk 15 above)
Comment (Luffy18346) [reply app/src/main/res/layout/administrator_controls_profile_view.xml:5]: I cannot remove for above mentioned reason

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt (hunk 18 above)
Comment (Luffy18346) [reply app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt:91]: Done

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt (hunk 17 above)
Comment (Luffy18346) [reply app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt]: Done

**Comment by rt4914** [comment]
@Luffy18346 Do not forget to assign PR back to me in `Assignees` section because otherwise there are very good chances that I might miss PR review.

File: app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAccountActionsViewModel.kt [hunk 19]
```diff
@@ -0,0 +1,4 @@
+package org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel
+
+/**  [ViewModel] for the recycler view in [AdministratorControlsFragment]. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAccountActionsViewModel.kt:3]: Extra space at start. 
`/** [ViewModel] for the recycler view in [AdministratorControlsFragment]. */`

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt [hunk 20]
```diff
@@ -82,6 +87,53 @@ class AdministratorControlsActivityTest {
...
+      onView(atPositionOnView(R.id.administrator_controls_list, 2, R.id.auto_update_topic_constraint_layout)).check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testAdministratorControlsFragment_loadFragment_displayApplicationSettings() {
+    ActivityScenario.launch<AdministratorControlsActivity>(createAdministratorControlsActivityIntent(0)).use {
+      onView(withId(R.id.administrator_controls_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(3))
+      onView(atPositionOnView(R.id.administrator_controls_list, 3, R.id.app_information_text_view)).check(matches(isDisplayed()))
+      onView(atPositionOnView(R.id.administrator_controls_list, 3, R.id.app_version_text_view)).check(matches(withText("App Version")))
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt:115]: Missed on these stings in this test case.

**Review by rt4914 — APPROVED** [review]
Looks Good thanks. There are some nit changes but I think it won't a nice thing to block this PR for another day on two nit changes. Instead I will merge this PR for now and Will fix those issues in the next PR right away to expedite the process.

Note to @Luffy18346  I am doing this considering the fact that I won't be able to review your PR tonight and that means you will get blocked on other work items till tomorrow.

File: app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAccountActionsViewModel.kt (hunk 19 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAccountActionsViewModel.kt:3]: Fixed in #740

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt (hunk 20 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt:115]: Fixed in #740
